src/components/mypage/SelectBuyerModal/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { IconButton, Divider, Avatar } from '@offer-ui/react'
+import type { ReactElement } from 'react'
+import { useState } from 'react'
+import { Styled } from './styled'
+import type { SelectBuyerModalProps } from './types'
+import { toLocaleCurrency } from '@utils'
+
+export const SelectBuyerModal = ({
+  productName,
+  buyers,
+  handleClickReviewButton,
```

Comment:
@price-offer/offer-fe 
컴포넌트 내부에서 핸들러 함수를 정의할 때, `handle` prefix를 붙이고
props로 전달할 때는 `on` prefix로 컨벤션 정의하는 건 어떠신가요?

```suggestion
  onClickReviewButton,
```

Reply:
좋심더 cc @chojooyoung

Reply:
[적용](https://github.com/price-offer/offer-fe/pull/107/commits/dbe833d84018b54c74b18926d7f84e1c03b5ba2d) 완룡